zammad/values.yaml

```diff
@@ -480,6 +480,14 @@ elasticsearch:
     heapSize: 512m
     masterOnly: false
     replicaCount: 1
+    resources: {}
```

Comment:
I don't see a `resources` top-level key in https://artifacthub.io/packages/helm/bitnami/elasticsearch, did I miss something?

Reply:
I added it  as `master.resources` here, not as top-level key.

Also see https://github.com/bitnami/charts/blob/main/bitnami/elasticsearch/values.yaml#L521

Reply:
Sorry, I missed that.